helm/funcx_endpoint/templates/endpoint-instance-config.yaml

```diff
@@ -20,9 +20,10 @@ data:
             HighThroughputExecutor(
                 max_workers_per_node={{ .Values.maxWorkersPerPod }},
                 address=address_by_route(),
-                strategy=KubeSimpleStrategy(max_idletime=3600),
+                strategy=KubeSimpleStrategy(max_idletime={{ .Values.maxIdleTime | default 3600 }}),
                 container_type='docker',
                 scheduler_mode='hard',
+                worker_debug={{ .Values.workerDebug }},
```

Comment:
There is confusing mismatch between helm (yaml) and the config file (python) regarding boolean values. Yaml wants lowercase `true` and `false` - The best thing is to keep the values.yaml file yaml, and use this to set the python config value

```
{{- ternary "True" "False" . Values.workerDebug }}
```

Reply:
Ah, great catch. Thanks!